topics/epigenetics/slides/introduction_atac.html

```diff
@@ -0,0 +1,121 @@
+---
+layout: introduction_slides
+logo: "GTN"
+video: true
+
+title: "ATAC-Seq data analysis"
+type: "introduction"
+questions:
+  - What is ATAC-Seq?
+  - What are the quality parameters to check for each dataset?
+  - How to analyse ATAC-Seq data?
+objectives:
+  - Understand ATAC-Seq
+  - Quality Parameters for ATAC-Seq
+  - Understand Peak calling for ATAC-Seq
+requirements:
+time_estimation: "30min"
+key_points:
+  - Run quality control on every sequencing dataset before any other analyses
+  - Choose QC parameters carefully
+  - Re-run FastQC to check the impact of the quality control
+contributors:
+  - heylf
+
+---
+
+# What is ATAC sequencing?
+
+---
+
+### Where my data comes from?
+
+![](../images/atac-seq/atac.jpg)
+
+<small>
+[*Buenrostro et al. 2013 Nat Methods*](https://doi.org/10.1002/0471142727.mb2129s109)
+</small>
+
+- Usage of hyperactive Tn5 transposase to insert sequencing adapters into open chromatin regions.
+- After adapter attachment the DNA is sheared by the Transposase itself.
+
+???
+
+It requires sequence-specific excision of a locus containing 19 base-pair inverted repeats.
+
+You have reads without nucleosomes (histones) = open chromatin regions, and reads with these complexes which will be filtered out because of their length.
+
+---
+
+### Characteristics of ATAC-Seq
+
+- Typically you have at least two biological replicates.
+- You also have a control. The control is the genomic DNA from the sample that is not treated with transposase. It is fragmented, e.g., by sonication. It has adapters ligated. It is sequenced along with the ATAC sample.
+- ATAC-Seq is paired-end sequencing. You need both ends to idenfiy a true open chromatin region. That is why you need both adapters.
+
+---
+
+## How to analyze ATAC-Seq data?
+
+---
+
+### Check the Insert Size
+
+![](../images/atac-seq/Screenshot_sizeDistribution_Good.png)
+
+
+- Typical insert size of 150-200 bp and then a multiple of the number of nucelosomes.
```

Comment:
This sentence is strange to me. I agree with the following 2.

Reply:
check